app/models/miq_schedule_worker/jobs.rb

```diff
@@ -14,7 +14,7 @@ def miq_server_worker_log_status
   end
 
   def miq_server_audit_managed_resources
-    queue_work(:class_name  => "MiqServer", :method_name => "audit_managed_resources", :queue_name => 'miq_server', :task_id => "audit_managed_resources", :server_guid => MiqServer.my_guid)
+    queue_work(:class_name  => "MiqServer", :method_name => "report_audit_details", :queue_name => 'miq_server', :task_id => "report_audit_details", :server_guid => MiqServer.my_guid)
```

Comment:
@Fryguy Updated the method name but I retained the job method name `miq_server_audit_managed_resources`, there is also a setting for the scheduler with a similar name https://github.com/ManageIQ/manageiq/blob/master/config/settings.yml#L1129, did you want these to be updated as well?

Reply:
Ugh yeah I forgot about that.  If we did rename that setting, then it would have to include a database migration.

Reply:
yeah thats partly why I didn't change the job method name, to keep some resemblance to the setting name. Also was trying to avoid the schema migration

Reply:
Of note, if you change the value going onto the queue, then the queue could contain links to methods that no longer exist. Not sure if that is a concern

Reply:
Yeah good point - probably need a minor data migration to handle that _or_ just let it fail.  There should be at most 1 on the queue, since it's run once daily.